### DISCUSSION TITLE
Non-Physical Products?

### DISCUSSION THREAD

**Discussion by mohamed--abdel-maksoud** [medusajs/medusa#2940] [Feature Requests]
Several stores sell digital products. For these we would need all the attributes, delivery settings, etc disabled.
I believe this simplifies the workflow and is potentially straightforward to implement.

Thanks in advance.

**Comment by jonebarker** [comment]
I have also been looking for any reference to digital product listing and protected download functionality in MedusaJS. My products are photoshop and 3D design files.

**Comment by meatandfat** [comment]
I saw this earlier https://github.com/medusajs/medusa/discussions/2228

**Reply by einar-hjortdal** [reply]
Shouldn't it be easy to make medusa support this by default? 

Medusa already has an S3 plugin, an S3 storage can store more than just images. Downloadable products are BLOBS just like images
